4337/test/eip4337/EIP4337Module.spec.ts

```diff
@@ -81,7 +81,7 @@ describe('Safe4337Module', () => {
         signature,
       })
 
-      await expect(safeModule.validateUserOp(userOp, ethers.ZeroHash, 0)).to.be.revertedWith('Unsupported execution function id')
+      await expect(entryPoint.executeUserOp(userOp, 0)).to.be.revertedWith('Unsupported execution function id')
```

Comment:
Why was this changed?

Reply:
As the modifier gets executed first, this will lead to getting the error `Unsupported entry point` instead of `Unsupported execution function id`.

Reply:
Another way to change this while keeping the test calling `validateUserOp` would be to call the `validateUserOp` function from the `entryPoint` contract.

I would slightly prefer that over calling `executeUserOp` which only implicitely calls `validateUserOp`

Reply:
Something like this will do the trick @nlordell 

```solidity=
const entryPointAddress = await ethers.getSigner(await entryPoint.getAddress())
const safeFromEntryPoint = safeModule.connect(entryPointAddress)
await expect(safeFromEntryPoint.validateUserOp.staticCall(userOp, ethers.ZeroHash, 0)).to.be.revertedWith('Unsupported execution function id')
```